Sources/Readability.swift

```diff
@@ -63,7 +64,38 @@ open class Readability {
 	let videoQueries: [(String, String?)] = [
 		("//head/meta[@property='og:video:url']", "content")
 	]
-
+    
+    let dateQueries: [(String, String?)] = [
+        ("//meta[@itemprop='datePublished']", "content"),
+        ("//span[@itemprop='datePublished']", "content"),
+        ("//head/meta[@property='article:published_time']","content"),
+        ("//head/meta[@name='article:published_time']","content"),
+        ("//head/meta[@name='sailthru.date']","content"),
+        ("//div[@class='date date--v2']", "data-datetime"),
+        ("//div[@class='keyvals']", "data-content_published_date"),
+        ("//form/input[@name='submit_time']", "value"),
+        ("//span/abbr[@class='published']", "title"),
+        ("//time[@itemprop='datePublished']", "datetime"),
+        ("//time[@class='entry-date']", "datetime"),
+        ("//time[@itemprop='datePublished']", "datetime"),
+        ("//time[@class='timestamp_article']", "datetime"),
+        ("//time[@class='timeago']", "datetime"),
+        ("//time[@class='post__date']", "datetime"),
+        ("//head/meta[@name='Date']", "content"),
+        ("//head/meta[@name='publish-date']", "content"),
+        ("//head/meta[@property='article:published_time']", "content"),
+        ("//head/meta[@name='date']", "content"),
+        ("/meta[@name='sailthru.date']", "content"),
+        ("//head/meta[@name='analyticsAttributes.articleDate']", "content"),
+        ("/meta[@property='og:article:published_time']", "content"),
+        ("//div[@class='companion-galleries embedded_story hasendslate galleries']", "data-published-date"),
+        ("//head/meta[@property='time']", "content"),
+        ("//time[@class='published-date hidden']", "datetime"),
+        ("//head/meta[@name='published_date']", "content"),
+        ("//head/meta[@property='og:title']", "content"),
```

Comment:
Are you sure we need this query for date parsing?

Reply:
Yes its tested in my project 
you can test by running demo project

Reply:
actually different sites use different date format and different tags for published date
some sites just use "published date "
some sites use "time"

Reply:
I mean this particular line `"//head/meta[@property='og:title']"` in array of queries :)

Reply:
no ... i removed it